screens/Home/MyVcs/DownloadingVcModalController.ts

```diff
@@ -1,13 +1,3 @@
-import { useSelector } from '@xstate/react';
-import { useContext } from 'react';
-import { selectVcLabel } from '../../../machines/settings';
-import { GlobalContext } from '../../../shared/GlobalContext';
-
 export function useDownloadingVcModal() {
```

Comment:
If this useDownloadingVCModel() is not used anywhere can we remove this ?

Reply:
Removed since it is not used anywhere